src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  background-color: #e5e5e5;
+  box-sizing: border-box;
+}
```

Comment:
lets add box-sizing fro all elements *
reset tag which u need like you did on the lesson with Ihor
move fonts to html selector